executor/executor_runner.h

```diff
@@ -27,6 +27,20 @@ inline std::ostream& operator<<(std::ostream& ss, const rpc::ExecRequestRawT& re
 		  << "\n";
 }
 
+inline void SoftExit()
+{
+	exitf("SYZ-EXECUTOR: PREEMPTED");
+}
+
+inline int AllocateProcID()
+{
+	static const int kMaxPids = 32; // matches prog.MaxPids
```

Comment:
As we discussed today, we need to also double-check that we do create shared devices/resources for all 32 procs.

Reply:
Added a commit for this. PTAL.